powerplant-android/src/main/java/org/kaaproject/kaa/examples/powerplant/PowerPlantAndroidDemoBuilder.java

```diff
@@ -56,12 +59,18 @@ protected void buildDemoApplicationImpl(AdminClient client) throws Exception {
 
         loginTenantDeveloper(client);
 
+        logger.info("Creating ctl schema...");
+
+        CTLSchemaDto ctlSchema = client.saveCTLSchemaWithAppToken(getResourceAsString("config_schema.avsc"), powerPlantAndroidApplciation.getTenantId(), powerPlantAndroidApplciation.getApplicationToken());
```

Comment:
it is better to create a protected method in AbstractDemoBuilder to avoid code duplication.
Smth like this:

CTLSchemaDto ctlSchema = saveCTLSchemaWithAppToken("config_schema.avsc", powerPlantAndroidApplciation);